src/Orc.Controls/Orc.Controls.Shared/Controls/DateTimePicker/ViewModels/DateTimePickerViewModel.cs

```diff
@@ -143,6 +145,24 @@ public int Second
                 Value = new DateTime(Year, Month, Day, Hour, Minute, value);
             }
         }
+
+        public string AmPm
+        {
+            get;
+            private set;
+        }
+        #endregion
+
+        #region Properties
+        protected override void OnPropertyChanged(AdvancedPropertyChangedEventArgs e)
+        {
+            base.OnPropertyChanged(e);
+
+            if (string.Equals(e.PropertyName, "Value"))
```

Comment:
There is no method HasPropertyChanged in AdvancedPropertyChangedEventArgs. Is it some extension ?